components/speedreader/features.cc

```diff
@@ -11,12 +11,15 @@
 namespace speedreader {
 
 const base::Feature kSpeedreaderFeature {
-      "Speedreader",
+  "Speedreader",
 #if BUILDFLAG(ENABLE_SPEEDREADER_FEATURE)
       base::FEATURE_ENABLED_BY_DEFAULT
 #else
       base::FEATURE_DISABLED_BY_DEFAULT
 #endif
 };
 
+const base::Feature kSpeedreaderReadabilityBackend{
```

Comment:
needs a space before {

Reply:
clang-format did this. i agree it looks kinda wonky, but it [happens in other files too ](https://github.com/brave/brave-core/blob/676d293dcb4dd21703fd0d5b8ff134e796f1410a/components/ntp_background_images/browser/features.cc#L15-L17)